src/context/actions/types.tsx

```diff
@@ -9,9 +9,11 @@ export type ActionMap<M extends { [index: string]: any }> = {
   }
 };
 
+
 export enum Types {
   Login_update = 'LOGIN_UPDATE',
-  Logout_update = 'LOGOUT_UPDATE'
+  Logout_update = 'LOGOUT_UPDATE',
```

Comment:
I highly suggest using `SCREAMING_SNAKE_CASE` for action types since that is the convention for constant values and this convention is widely adopted by the Redux, Context and other state management communities.